scripts/qesap/test/e2e/main_local.tf

```diff
@@ -0,0 +1,4 @@
+resource "local_file" "foo" {
```

Comment:
This is the most cool stuff, it allow to run e2e tests without any cloud credential but simulating resource creation and delete